vocabularies/quality.ttl

```diff
@@ -0,0 +1,685 @@
+  skos:inScheme dq-iso-19157:Dimensions ;
+  skos:prefLabel "Thematic Accuracy"@en ;
+.
+dq-iso-19157:ThematicClassificationCorrectness
+  a dqv:Dimension ; a skos:Concept ;
+  rdfs:label "ThematicClassificationCorrectness"@en ;
+  skos:broader dq-iso-19157:ThematicAccuracy ;
+  skos:definition "Comparison of the classes assigned to features or their attributes to a universe of discourse"@en ;
+  skos:inScheme dq-iso-19157:Dimensions ;
+  skos:prefLabel "Thematic Classification Correctness"@en ;
+.
+dq-iso-19157:TopologicalConsistency
+  a dqv:Dimension ; a skos:Concept ;
+  rdfs:label "TopologicalConsistency"@en ;
+  skos:broader dq-iso-19157:LogicalConsistency ;
+  skos:definition "Correctness of the explicitly encoded topological characteristics of the data set as described by the scope"@en ;
+  skos:inScheme dq-iso-19157:Dimensions ;
+  skos:prefLabel "Topological Consistency"@en ;
+.
+dq-iso-19157:UsabilityElement
+  a dqv:Dimension ; a skos:Concept ;
+  rdfs:label "UsabilityElement"@en ;
+  skos:definition "Degree of adherence of a data set to a specific set of requirements"@en ;
+  skos:inScheme dq-iso-19157:Dimensions ;
+  skos:prefLabel "Usability Element"@en ;
+.
+dq-iso-25012:Dimensions
+  a skos:ConceptScheme ;
+  rdfs:label "Dimensions of data quality (ISO 25012)"@en ;
+  skos:prefLabel "Dimensions of data quality (ISO 25012)"@en ;
+  dcterms:created "2020-11-16"^^xsd:date ;
+dcterms:creator <http://linked.data.gov.au/def/gsq-alias/des> ;
+dcterms:modified "2020-11-16"^^xsd:date ;
+dcterms:publisher <http://linked.data.gov.au/def/gsq-alias/des> ;
+  skos:hasTopConcept dq-iso-25012:inherentDataQuality ;
+  skos:hasTopConcept dq-iso-25012:inherentSystemDependentDataQuality ;
+  skos:hasTopConcept dq-iso-25012:systemDependentDataQuality ;
+.
+dq-iso-25012:accessibility
+  a dqv:Dimension ; a skos:Concept ;
+  dcterms:source <https://www.w3.org/TR/vocab-dqv/#DimentsionsMetricsHints> ;
+  rdfs:label "Accessibility"@en ;
+  skos:definition "The degree to which data can be accessed in a specific context of use, particularly by people who need supporting technology or special configuration because of some disability."@en ;
+  skos:inScheme dq-iso-25012:Dimensions ;
+  skos:prefLabel "Accessibility"@en ;
+  dqv:inCategory dq-iso-25012:inherentSystemDependentDataQuality ;
+  skos:broader dq-iso-25012:inherentSystemDependentDataQuality ;
+.
+dq-iso-25012:accuracy
+  a dqv:Dimension ; a skos:Concept ;
+  dcterms:source <https://www.w3.org/TR/vocab-dqv/#DimentsionsMetricsHints> ;
+  rdfs:label "Accuracy"@en ;
+  skos:definition "The degree to which data has attributes that correctly represent the true value of the intended attribute of a concept or event in a specific context of use."@en ;
+  skos:inScheme dq-iso-25012:Dimensions ;
+  skos:prefLabel "Accuracy"@en ;
+  dqv:inCategory dq-iso-25012:inherentDataQuality ;
+  skos:broader dq-iso-25012:inherentDataQuality ;

```

Comment:
There are two concepts called Acuracy.
These should be amalgamated if they are the same, or given unique labels that distinguish them from each other

Reply:
Refer to the comments above regarding the term 'Completeness'